src/XamlX/Parsers/SystemXamlMarkupExtensionParser/MeScanner.cs

```diff
@@ -61,20 +62,20 @@ internal partial class MeScanner
 
```

Comment:
For context, this class was ported from System.XAML, but I just double checked - original wasn't ported to nullability. Unfortunately.
https://github.com/dotnet/wpf/blob/c30fa6e62d3b94e5c65dda316f5c04397a8cb70d/src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Parser/MeScanner.cs#L41